Cargo.toml

```diff
@@ -23,17 +23,16 @@ http = "1.1.0"
 # This is currently used in the jwt_vp test to go from a `VeriableCredential` to an `AnyJsonCredential` type.
 # There may be a better way to handle this that doesn't require the `json-syntax` crate directly.
 json-syntax = { version = "0.12.5", features = ["serde_json"] }
-jsonschema = "0.18.0"
+jsonschema = { git = "https://github.com/spruceid/jsonschema.git", branch = "feat/wasm-support" }
```

Comment:
Are you planning on raising a PR with the changes?

Reply:
Ideally, yes, but receiving a `Pull request creation failed. Validation failed: must be a collaborator` error when opening a PR against the origin repository. We can merge this branch into main, but likely will need to reach out to the repo owner and allow him to accept changes.